examples/Accelerators/DarcyAccelerated/darcy_accelerated.jl

```diff
@@ -0,0 +1,193 @@
+    darcy = Setup_Param(pts_per_dim, obs_ΔN, κ_true)
+    h_2d = solve_Darcy_2D(darcy, κ_true)
+    y_noiseless = compute_obs(darcy, h_2d)
+    obs_noise_cov = 0.05^2 * I(length(y_noiseless)) * (maximum(y_noiseless) - minimum(y_noiseless))
+    truth_sample = vec(y_noiseless + rand(rng, MvNormal(zeros(length(y_noiseless)), obs_noise_cov)))
+
+    # Now we set up the Bayesian inversion algorithm. The prior we have already defined to construct our truth
+    prior = pd
+
+    # We define some algorithm parameters, here we take ensemble members larger than the dimension of the parameter space
+    N_ens = dofs + 2    # number of ensemble members
+    N_iter = 20         # number of EKI iterations
+    N_trials = 5       # number of trials 
+
+    """
+    Define function run_darcy() with parameter ACCELERATED to solve the Darcy problem using either
+    the default EKI algorithm (ACCELERATED=TRUE) or the EKI algorithm with Nesterov acceleration (ACCELERATED=FALSE).
+
+    Returns a vector of errors evaluated after each EKI iteration.
+    """
+    function run_darcy(ACCELERATED::Bool)
+        errs = zeros(N_trials, N_iter)
+
+        # can make separate RNG for each thread, run with multithreading
+
+        for trial in 1:N_trials
+            if USE_SCHEDULER
+                scheduler = scheduler_def
+            else
+                scheduler = DefaultScheduler(0.1)
+            end
+
+            # We sample the initial ensemble from the prior, and create the EKP object as an EKI algorithm using the `Inversion()` keyword
+            initial_params = construct_initial_ensemble(rng, prior, N_ens)
+            if ACCELERATED
+                ekiobj = EKP.EnsembleKalmanProcess(
+                    initial_params,
+                    truth_sample,
+                    obs_noise_cov,
+                    Inversion(),
+                    accelerator = NesterovAccelerator(),
+                    scheduler = deepcopy(scheduler),
+                )
+            else
+                ekiobj = EKP.EnsembleKalmanProcess(
+                    initial_params,
+                    truth_sample,
+                    obs_noise_cov,
+                    Inversion(),
+                    scheduler = deepcopy(scheduler),
+                )
+            end
+
+            # Run the EKI algorithm using EKP functions, recording parameter error after each iteration.
+            err = zeros(N_iter)
+            for i in 1:N_iter
+                params_i = get_ϕ_final(prior, ekiobj)
+                g_ens = run_G_ensemble(darcy, params_i)
+                EKP.update_ensemble!(ekiobj, g_ens, deterministic_forward_map = true)
+                err[i] = get_error(ekiobj)[end]
+                errs[trial, :] = err
+            end
+
+        end
+        return errs
+    end
+
+    errs_trad = log.(run_darcy(false))
```

Comment:
In the other examples you compare trad, acc and acc_cs. 
Could you make Darcy compare all three please (with consistent graph colors (i.e. these two curves should be black and blue, and the constant step one you add should be red)